Vagrantfile

```diff
@@ -166,11 +175,17 @@ Vagrant.configure("2") do |config|
         config.ignition.hostname = vm_name
         config.ignition.config_vmdk = File.join(File.dirname(__FILE__), "config" + i.to_s  + ".vmdk")
         config.ignition.config_img = "config" + i.to_s  + ".img"
-
         # when the ignition config doesn't exist, the plugin automatically generates a very basic Ignition with the ssh key
-        # and previously specified options (ip and hostname)
+        # and previously specified options (ip and hostname). Otherwise, it appends those to the provided config.ign below
         if File.exist?(IGNITION_CONFIG_PATH)
-          config.ignition.path = IGNITION_CONFIG_PATH
+          data = File.read(IGNITION_CONFIG_PATH)
```

Comment:
I'm not a fan of all of this substitution. We should instead teach coreos-metadata how to get a hold of this information so CT may be used to configure the cluster. I think this entire commit can be dropped in favor of the coreos-metadata functionality.